audb/core/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import audb
+
+
+@pytest.fixture(autouse=True)
+def add_audb_with_public_data(doctest_namespace):
```

Comment:
Why do we need this now?

Reply:
For the docstring test we depend on `emodb` as we do for building the documentation, but in the other tests we just use locally published data. I cannot simply add the public repo to all the other tests as some of them depend on a fixed number of available databases in the repo, and the public repo can of course grow over time.

Reply:
Ah now I get it! And  `doctest_namespace` automatically applies the fixture to `docstring` examples? Sweet!

Reply:
Yes, I added now a docstring to this function which explains why we need it.

Reply:
> And doctest_namespace automatically applies the fixture to docstring examples?
Yes, but the file has to be in the same folder as the code files.

Oh, I just realized I have a mistake in the docstring, will update it.

Reply:
OK, fixed it.